README.md

```diff
@@ -258,10 +258,37 @@ Once you have completed any given exercise make sure to compare your implementat
 
 Hosted by tech.io, this [SYCL Introduction](https://tech.io/playgrounds/48226/introduction-to-sycl/introduction-to-sycl-2) tutorial introduces the concepts of SYCL. The website also provides the ability to compile and execute SYCL code from your web browser.
 
-SYCL and the SYCL logo are trademarks of the Khronos Group Inc.
+## Connecting to DevCloud via SSH
+
+* Start by creating an [Intel DevCloud account][intel-devcloud] account if you do not already have one and login in.
+* Initialize the SSH configuration by clicking on [Automated Configuration](https://devcloud.intel.com/oneapi/documentation/connect-with-ssh-linux-macos/) and follow the instructions to setup the SSH configuration file.
+* SSH into DevCloud (```ssh devcloud```)
+
+You are now ready to start with the first [lesson][lesson-1-slides]. Enjoy !
+
```

Comment:
I would still like to see instructions for accessing a Terminal through the Jupyter notebook access as this is the simplest for some people. There is a GUI for editing files and a full terminal window as well. One issue is that this seems to be moved around sometimes, but currently it is on this page https://console.cloud.intel.com/training. Maybe this is too hard to keep up to date though?